modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/ActivationKeysTable/components/TableKeyDetails/index.jsx

```diff
@@ -47,7 +48,7 @@ const TableKeyDetails = ({currentActivationKey, setValueToCopyToClipboard}) => {
 		currentActivationKey?.expirationDate
 	);
 
-	const formatedProductName = getFormatedProductName(
+	const formattedProductName = getFormatedProductName(
```

Comment:
can you change the function name to be spelled correctly as well. `getFormattedProductName`

Reply:
Done.